public/js/populateDropdowns.js

```diff
@@ -100,6 +100,31 @@ const checkedCohortDates = () => {
   })
 }
 
+  let outputAssessmentDates = (dates, el = null) => {
+    const element = el || document.querySelector('#assessmentDay');
+    let assessmentOptions = '';
+
+    if (element) {
+      dates.forEach(item => {
+        assessmentOptions += '<option ';
+        if (element.dataset.selected === item.date) {
+          assessmentOptions += 'selected ';
+        }
+        let date = new Date(item.date);
+        let dateOptions = {year: 'numeric', month: 'long', day: 'numeric'};
+        assessmentOptions += `value="${
+            item.id
+        }">${date.toLocaleDateString(
+            'en-GB',
+            dateOptions
+        )}</option>`;
+      });
+
+      element.innerHTML += assessmentOptions;
```

Comment:
there is no unselected assessment option. What if we want to edit an applicant and not set an assessment date yet?